rundeck-data-model/src/main/java/org/rundeck/app/data/providers/v1/WebhookDataProvider.java

```diff
@@ -0,0 +1,43 @@
+package org.rundeck.app.data.providers.v1;
+
+import org.rundeck.app.data.model.v1.webhook.RdWebhook;
+import org.rundeck.app.data.model.v1.webhook.dto.SaveWebhookRequest;
+import org.rundeck.app.data.model.v1.webhook.dto.SaveWebhookResponse;
+
+import java.util.List;
+
+public interface WebhookDataProvider extends DataProvider {
+    RdWebhook get(Long id);
+    RdWebhook getWebhook(Long id);
+    RdWebhook getWebhookByToken(String token);
+    RdWebhook getWebhookWithProject(Long id, String project);
+    RdWebhook getWebhookByUuid(String uuid);
+    /**
+     * Retrieves a new instance of RdWebhook, not stored on the database
+     *
+     * @return Webhook instance
+     */
+    RdWebhook buildWebhook();
```

Comment:
There doesn't seem to be a clear purpose for putting this on the data provider. What is the advantage of creating a new object this way?

Reply:
I removed it